src/components/screens/profile/ProfileControlledInput.tsx

```diff
@@ -0,0 +1,44 @@
+import { ControlledField } from '@baca/components'
+import { isWeb } from '@baca/constants'
+import { Text, Box } from '@baca/design-system'
+import { useWeb } from '@baca/hooks'
+import { ProfileControlledInputProps } from '@baca/types/ProfileInputProps'
+
+export const ProfileControlledInput = ({
+  label,
+  name,
+  placeholder,
+  control,
+  errors,
+  isDisabled = false,
+  onFocus,
+  onSubmitEditing,
+}: ProfileControlledInputProps) => {
+  const { shouldApplyMobileStyles } = useWeb()
+
+  return (
+    <Box
+      justifyContent="space-between"
+      flexDirection={isWeb && !shouldApplyMobileStyles ? 'row' : 'column'}
+      mb={isWeb ? 10 : 0}
+      maxW={800}
+    >
+      <Text.SmBold flex={1}>{label}</Text.SmBold>
+      <Box flex={isWeb ? 2 : 0}>
+        <ControlledField.Input
+          control={control}
```

Comment:
most of the props could be passed as `...rest` to be more readable ;)

Reply:
```
export const ProfileControlledInput = ({
  name,
  isDisabled = false,
...rest
}: ProfileControlledInputProps) => {
```
...

```  
<ControlledField.Input
          autoCapitalize="none"
          inputMode={name === 'email' ? 'email' : 'text'}
          name={name}
          testID={`${name}Input`}
          isDisabled={isDisabled}
          {...(!isWeb && { label })}
          {...rest}
        />
```

Reply:
Nice to use rest operator, helps significantly while reading code.  @micbaumr, I've added all the changes to:

> feat/profile-image-section

In order to avoid git conflicts 🤠